azkaban-exec-server/src/test/java/azkaban/execapp/FlowPreparerTest.java

```diff
@@ -183,4 +184,37 @@ public void testProjectCacheDirCleaner() throws IOException, InterruptedExceptio
     assertThat(this.projectsDir.listFiles()).containsExactlyInAnyOrder(expectedRemainingFiles
         .toArray(new File[expectedRemainingFiles.size()]));
   }
+
+  @Test
+  public void testProjectsCacheMetricsZeroHit() {
+    //given
+    final FlowPreparer.ProjectsDirCacheMetrics cacheMetrics = new ProjectsDirCacheMetrics();
+
+    //when zero hit and zero miss then
+    assertThat(cacheMetrics.getHitRatio()).isEqualTo(0);
+
+    //when
+    cacheMetrics.incrementCacheMiss();
+    //then
+    assertThat(cacheMetrics.getHitRatio()).isEqualTo(0);
+
+  }
+
+  @Test
+  public void testProjectsCacheMetricsHit() {
+    //given
```

Comment:
The comments in the test cases are not quite informative. Consider either adding more details or removing redundant ones?

Reply:
updated thanks